lib/rules/disallow-keywords-in-comments.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Disallows keywords in your comments, such as TODO or FIXME
+ *
+ * Type: `Boolean` or `String` or `Array`
+ *
+ * Values:
+ * - `true`
+ * - `'\b(word1|word2)\b'`
+ * - `['word1', 'word2']`
+ *
+ * #### Examples
+ *
+ * ```js
+ * "disallowKeywordsInComments": true
+ * "disallowKeywordsInComments": '\b(word1|word2)\b'
```

Comment:
I'm not sure, but feels like it should be `double-slash b` (`"\\b(...)\\b"`) and in double-quotes ('cause of JSON). Right?